R/load_files.R

```diff
@@ -1437,20 +1438,22 @@ get_EnVision_properties <- function(results.list, fname) {
 #' @details
 #' All plate sizes assume 1.5x nrows = ncolumns.
 #' @keywords internal
-#' 
+#'
 #' @return charvec with plate dims
-#' 
+#'
 .get_plate_size <- function(df) {
-  n_col <-
-    1.5 * 2 ^ ceiling(log2((ncol(df) - 2) / 1.5))
-  n_row <- n_col / 1.5
+  n_col <- min(
```

Comment:
I think it should be well tested to be sure it does not disrupt different plate sizes